editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1399,6 +1430,8 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 void SpriteFramesEditor::_update_library_impl() {
 	pending_update = false;
 
+	// WARN_PRINT("YET ANOTHER TEST"); // THIS DOES RUN
+
```

Comment:
Debug code that should be removed I assume?

Reply:
whoops. removed